### PR TITLE
Mobile-desktop popup

### PR DESCRIPTION
**Pull request by Elhambasir**
Popup window is done for mobile and desktop versions.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -655,6 +687,221 @@ footer {
...
+  width: 97%;
+  height: fit-content;
+  position: absolute;
+  top: 0;
+  left: 0;
+  margin: 0 auto;
+  float: left;
+  padding: 3% 1.5%;
+  display: none;
+}
```
Comment (christianbiring1) [inline style.css]: - [ ] Great job on rendering the projects list dynamically :+1: However, Kindly consider making sure when the pop modal is activated the user can't scroll anymore to overpass it as is the case on your design.
Hint: `to fix this issue, you may consider adding a container  for the pop-up in the HTML doc, give it the absolute position according to the body element, and center it as you did for the pop up and I hope from there you will be good to go.`

### Your design :point_down: 
image
image

File: style.css [hunk 2]
```diff
@@ -655,6 +687,221 @@ footer {
...
+
+.popup .popup_card .project_title:nth-child(1) {
+  flex: 1;
+  text-align: left;
+  cursor: pointer;
+}
+
+.popup .popup_card .project_title:nth-child(2) {
+  flex: 1;
+  text-align: right;
```
Comment (christianbiring1) [inline style.css:747]: - [ ] Kindly make sure you stick to the design as much as possible, the spacing(margin, padding) should suppose to be similar to the Figma design.

### Your design :point_down: 
image
### 2
image


### Figma template :point_down: 
image

### 2

image

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @Elhambasir @Nelofarzabi , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
soclose

## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title`
- [ ] The projects are rendered dynamically :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] `Telling in detail in the description box a descriptive summary of what changes you have implemented in the PR.`

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by frank1738 — APPROVED** [review]
Hi @Elhambasir , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ Good commit messeges

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**